docs/rbe-setup: fix wrong usage of action_env (#7451)
buildbuddy-io/buildbuddy: docs/rbe-setup.md

[Bazel docs](https://docs.bazel.build/versions/master/command-line-reference.html#flag--incompatible_strict_action_env)

### --action_env, --test_env

You can set environment variables that are available to actions with the `--action_env` flag.
If you want to set environment variables that are only available to test actions, you can use the `--test_env` flag.

> This is commonly used to set ennvironment variables such as `GO_TEST_WRAP_TESTV=1`, which modify how `go_test` rule handles the XML reports.

Reviewer: ```suggestion
This is commonly used to set environment variables such as `GO_TEST_WRAP_TESTV=1`, which modifies how the `go_test` rule handles the XML reports.
```

```bash
--test_env=GO_TEST_WRAP_TESTV=1
```

[`--action_env` Bazel docs](https://docs.bazel.build/versions/master/command-line-reference.html#flag--action_env)
[`--test_env` Bazel docs](https://docs.bazel.build/versions/master/command-line-reference.html#flag--test_env)

### --repo_env

> This flag set additional environment variables that are available to repository rules executions.

Reviewer: ```suggestion
This flag sets additional environment variables that are available to repository rules executions.
```

> It's commonly used to set environment variables that are used to augment how toolchains and external repositories are prepared before the build phase.

Reviewer: Maybe drop `before the build phase` as Bazel no longer has cleanly separated phases.

Author: ```suggestion
It's commonly used to set environment variables that are used to augment how toolchains and external repositories are prepared.
```
